routing.py

```diff
@@ -0,0 +1,40 @@
+import urllib.parse as urlparse
+from typing import Tuple
+
+from domain.metric_type import MetricType
+from domain.types import AgentID
+
+
+def encode_matrix_path(metric: MetricType) -> str:
+    if metric == MetricType.PACKET_LOSS:
+        return "/packet_loss"
+    if metric == MetricType.JITTER:
+        return "/jitter"
+    if metric == MetricType.LATENCY:
+        return "/latency"
+    return "/"
+
+
+def encode_chart_path(from_agent, to_agent: AgentID, return_path: str) -> str:
+    return f"/chart?from={from_agent}&to={to_agent}&backto={return_path}"
```

Comment:
Will the "backto" be visible in the history?

Reply:
Removing back button together with backto, so - no :)